lib/control-app.js

```diff
@@ -5,6 +5,10 @@ var express = require('express');
 var expstate = require('express-state');
 var browserify = require('browserify');
 var im = require('istanbul-middleware');
+var watchify = require('watchify');
+var _ = require('lodash');
```

Comment:
I just stick to what was done in the actual browserify builder for example. Did not want to generate more noise in the PR. Mind doing it (me) in another PR after this one?